src/styles/stars.css

```
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  }
+
+  .stars__star {
+    justify-content: space-between;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-size: auto;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  .stars--1 :nth-child(-n+1) {
```

```suggestion
  .stars--1 :nth-child(-n+1),
  .stars--2 :nth-child(-n+2),
  ... {
```
Consider using commas to group selectors and avoid repeating code